docs/docs/user-guide/access-control.md

```diff
@@ -0,0 +1,16 @@
+# Overview of Access Control in Mathesar
```

Comment:
I'm a bit concerned that "Access Control" is too jargon-ey. Perhaps "Roles & Permissions" could work?

Reply:
Thanks for explaining! I think "Access Control" makes perfect sense then as a phrase to encompass all aspects of, well, access control in Mathesar. Any future language optimization here could be downstream of changes to the broader conceptual model of access control in the app,as you mentioned.